src/TextArea/TextArea.stories.jsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import TextArea from './index';
+
+storiesOf('Textarea', module)
+  .add('minimal usage', () => (
+    <TextArea
+      name="name"
+      label="First Name"
+      value="Foo Bar"
+    />
+  ))
+  .add('scrollable', () => (
+    <TextArea
+      name="name"
+      label="Information"
+      value="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+       incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+       exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure
+       dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+       Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+       mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+       sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+       quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
+       irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+       Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+       mollit anim id est laborum."
+    />
+  ))
+  .add('validation', () => (
+    <TextArea
+      name="username"
+      label="Username"
+      description="The unique name that identifies you throughout the site."
+      validator={(value) => {
+        let feedback = { isValid: true };
```

Comment:
Not a big deal but why not
```javascript
if (value.length < 3) {
  return { }
}

return { }
```
I get the single exit point argument but it does seem a lot simpler than dealing with an (IMHO) unnecessary variable assignment.